### ISSUE TITLE
 update_tmserver: propose to run command with --rebuild flag

### ISSUE THREAD

**Issue by Mixaill** [translate/pootle#3948]
Ubuntu 15.04
Elasticsearch 1.7.1

When trying to populate Local TM first time with *update_tmserver* without *--rebuild* flag
```
(env)pootle@alles:~/pootle/pootle$ python manage.py update_tmserver
TransportError(400, u'SearchPhaseExecutionException[Failed to execute phase [query], all shards failed; shardFailures {[_y4FTYjbTNuNUGHTJs-kew][translations][0]: SearchParseException[[translations][0]: query[ConstantScore(*:*)],from[-1],size[-1]: Parse Failure [Failed to parse source [{"query": {"match_all": {}}, "facets": {"stat1": {"statistical": {"field": "revision"}}}}]]]; nested: FacetPhaseExecutionException[Facet [stat1]: No mapping found for field [revision]]; }{[_y4FTYjbTNuNUGHTJs-kew][translations][1]: SearchParseException[[translations][1]: query[ConstantScore(*:*)],from[-1],size[-1]: Parse Failure [Failed to parse source [{"query": {"match_all": {}}, "facets": {"stat1": {"statistical": {"field": "revision"}}}}]]]; nested: FacetPhaseExecutionException[Facet [stat1]: No mapping found for field [revision]]; }{[_y4FTYjbTNuNUGHTJs-kew][translations][2]: SearchParseException[[translations][2]: query[ConstantScore(*:*)],from[-1],size[-1]: Parse Failure [Failed to parse source [{"query": {"match_all": {}}, "facets": {"stat1": {"statistical": {"field": "revision"}}}}]]]; nested: FacetPhaseExecutionException[Facet [stat1]: No mapping found for field [revision]]; }{[_y4FTYjbTNuNUGHTJs-kew][translations][3]: SearchParseException[[translations][3]: query[ConstantScore(*:*)],from[-1],size[-1]: Parse Failure [Failed to parse source [{"query": {"match_all": {}}, "facets": {"stat1": {"statistical": {"field": "revision"}}}}]]]; nested: FacetPhaseExecutionException[Facet [stat1]: No mapping found for field [revision]]; }{[_y4FTYjbTNuNUGHTJs-kew][translations][4]: SearchParseException[[translations][4]: query[ConstantScore(*:*)],from[-1],size[-1]: Parse Failure [Failed to parse source [{"query": {"match_all": {}}, "facets": {"stat1": {"statistical": {"field": "revision"}}}}]]]; nested: FacetPhaseExecutionException[Facet [stat1]: No mapping found for field [revision]]; }]')
```

It would be nice if Pootle will propose to use *--rebuild* flag.

There is error in docs:  http://pootle.readthedocs.org/en/latest/features/translation_memory.html#elasticsearch-based-local-tm

command should be with *--rebuild*.

**Comment by julen** [comment]
Let's hear from @dwaynebailey when he is back for the currently expected behavior.

**Comment by dwaynebailey** [comment]
@Mixaill sorry, didn't read this report carfully enough.  Added a ``--rebuild`` to the first ``update_tmserver`` instruction.  I don't think its the full fix, I'll leave #4120 for that. But this should catch other unlucky people setting up LocalTM
